samples/BikeSharingApp/BikeSharingWeb/pages/preview.js

```diff
@@ -149,7 +151,7 @@ class PreviewBase extends Component {
 
                                 <MediaQuery minWidth={600}>
                                     <div className="divider">
-                                        <FormButton primary onClick={this.handleClick.bind(this)}>Rent bike</FormButton>
+                                        <FormButton primary disabled={!this.state.isLoaded} disabled={!this.state.isLoaded} onClick={this.handleClick.bind(this)}>Rent bike</FormButton>
```

Comment:
Typo? I don't think you need to pass the `disabled` prop twice here. Same for line 168 as well.

Reply:
Oh, great point. Will make this change when I update the image tag. Thanks Logan!

Reply:
I agree, nice catch!